### PR TITLE
Add unidecode recipe

### PR DESCRIPTION
**Pull request by synapticarbors**

### CODE REVIEW

File: recipes/unidecode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - unidecode
+  commands:
+    - unidecode --help
+
+about:
+  home: https://pypi.python.org/pypi/Unidecode
+  license: GNU General Public License v2 or later (GPLv2+)
```
Comment (jakirkham) [inline recipes/unidecode/meta.yaml]: `GPL-2` is sufficient.

Comment (jakirkham) [inline recipes/unidecode/meta.yaml]: Also please add license file.

```
  license_file: LICENSE
```

File: recipes/unidecode/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipes/unidecode/meta.yaml]: Please add [`entry_points`]( http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points ).

```yaml
  entry_points:
    - unidecode = unidecode.util:main
```

File: recipes/unidecode/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/unidecode/meta.yaml]: We should be able to drop this after adding the entrypoints above.

**Comment by msarahan** [comment]
Thanks, merging.  Please ping the source owners about being maintainers here if they'd like in an issue on the feedstock.

**Pull request merged**